### ISSUE TITLE
Issue with domtastic@0.15.3

### ISSUE THREAD

**Issue by medialwerk** [webpro/DOMtastic#88]
Hello,

i load domtastic@0.15.3 with webpack@3.8.1 and chrome. The following error occurs:

```
trigger.js:169 Uncaught TypeError: CustomEvent is not a constructor
 at isEventBubblingInDetachedTree (trigger.js:169)
 at Object.defineProperty.value (trigger.js:172)
 at __webpack_require__ (bootstrap 8917c05722c78281add6:678)
 at fn (bootstrap 8917c05722c78281add6:88)
 at Object.defineProperty.value (index.js:1)
 at __webpack_require__ (bootstrap 8917c05722c78281add6:678)
 at fn (bootstrap 8917c05722c78281add6:88)
 at Object.<anonymous> (_math-scale.js:18)
 at __webpack_require__ (bootstrap 8917c05722c78281add6:678)
 at __webpack_exports__.a (bootstrap 8917c05722c78281add6:724)
```

Is that a known issue at the moment? - With domtastic@0.14.0 is everything fine.

Thanks!

**Comment by webpro** [comment]
Please see #69. Not sure what field(s) in `package.json` should be used these days. DOMtastic currently uses this:

```
  "main": "dist/domtastic.js",
  "jsnext:main": "src/index.js",
  "module": "src/index.js"
```

**Comment by nickngqs** [comment]
I'm also having this issue. I'm using rollup to bundle my files. Since its ES6, i suppose it needs to be transpile to ES5? But my rollup.config has babel ignore node_modules. Is this maybe the reason or?

**Comment by webpro** [comment]
I'd like to support both bundlers, but I do need some example setup or reproducible case to work/test with.

**Comment by webpro** [comment]
I think I've fixed this in [v0.15.4](https://github.com/webpro/DOMtastic/releases/tag/0.15.4).

**Comment by medialwerk** [comment]
Thank you very much!
